### DISCUSSION TITLE
Login went wrong. Please try again.

### DISCUSSION THREAD

**Discussion by Kosnstantin** [saleor/saleor#14759] [Q&A]
Hello everyone. I registered with my google acc on Saleor, next i install dashboard in my folder, next i wrote in console:
"npm run dev" and after that i went to http://localhost:9000/
I enter my login and password, it responds that the login is invalid, although the account on the site (not locally) logs in normally

**Comment by yajuve** [comment]
See https://github.com/saleor/saleor-platform/issues/298#issuecomment-2100315428
